dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -637,6 +637,15 @@ Resources:
       CidrBlock: 172.31.1.0/27
       MapPublicIpOnLaunch: true
 
+  VPCPrivateSubnet1:
```

Comment:
Super minor, but is there any reason for those 1s at the ends?

Reply:
No reason, just removed numbering 👍